From 6d67e98e017dcd65dc33f23f5549a5fb23ffe15d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 27 Mar 2012 12:31:55 +0000
Subject: [PATCH 082/462] mcorba.lib, mcorbas.lib V 1.0.6.18 cannot use bugfix
 with vc6 (TAO1.4 does not support ORB.set_default) revert to old version for
 vc6

git-svn-id: svn://localhost/SelfServiceBuild/trunk@110 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index b753a9d..0457481 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -20,7 +20,7 @@ forcebuild:
 # DIRECTORIES
 ######################################################################
 
-!if "$(VC_COMPILER_VERSION)" == "VC6" 
+!if "$(VC_COMPILER_VC6)" == "6" 
 CORBA_DIRECTORIES = \
   Massai\cpp\MassaiCorba_vc6\dll \
   Massai\cpp\MassaiCorba_vc6\static
@@ -29,7 +29,7 @@ CORBA_DIRECTORIES = \
   Massai\cpp\MassaiCorba\dll \
   Massai\cpp\MassaiCorba\static
 !endif
-!if "$(VC_COMPILER_VERSION)" == "VC6" 
+!if "$(VC_COMPILER_VC6)" == "6" 
 TRACE_DIRECTORIES = \
  Services\cpp\TrcServ \
  Services\cpp\TrcServ\static
-- 
2.41.0.windows.1

